bd_spot_wrapper/spot_wrapper/keyboard_teleop.py

```diff
@@ -23,18 +24,24 @@
 
 # Where the gripper goes to upon initialization
 INITIAL_POINT = np.array([0.5, 0.0, 0.35])
-INITIAL_RPY = np.deg2rad([0.0, 45.0, 0.0])
+INITIAL_RPY = np.deg2rad([0.0, np.pi / 4, 0.0])
```

Comment:
HI Kavit, it seems that you want to have pi/4 rad = 45 deg, But there is a np.deg2rad() outside the list

Reply:
Makes sense, good find.
Addressed.